socorro/processor/rules/base.py

```diff
@@ -4,24 +4,21 @@
 
 import logging
 
-from configman import RequiredConfig
 import markus
 
 
 metrics = markus.get_metrics('processor.rule')
 
 
-class Rule(RequiredConfig):
+class Rule:
     """Base class for transform rules
 
     Provides structure for calling rules during the processor pipeline and also
     has some useful utilities for rules.
 
     """
 
-    def __init__(self, config=None, quit_check_callback=None):
-        self.config = config
-        self.quit_check_callback = quit_check_callback
```

Comment:
This would take a `quit_check_callback`, but none of the rules used it, so I'm nixing it.